src/lib/journey/journey.store.ts

```diff
@@ -37,10 +37,8 @@ export function initialize(initOptions?: StepOptions): JourneyStore {
      * form failure due to 400 response from ForgeRock.
      */
     let previousCallbacks: FRCallback[] | undefined;
-    let previousStage: string | undefined;
 
     if (prevStep && prevStep.type === StepType.Step) {
-      previousStage = prevStep?.getStage && prevStep.getStage();
```

Comment:
I don't have a problem with it but am a little curious - why are we removing the reference to `previousStage` and what were we doing with it before this PR?

Reply:
This was vestigial code that should have been removed a while ago, but it was left behind. I used to use the step's stage value to do a comparison for detecting if the user encountered some kind of validation issue, like in registration. But, a better method replaced it quite a while ago.